exporter.go

```diff
@@ -117,23 +118,38 @@ func (c *GaugeContainer) Get(metricName string, labels prometheus.Labels, help s
 
 type SummaryContainer struct {
 	Elements map[uint64]prometheus.Summary
+	mapper   *metricMapper
 }
 
-func NewSummaryContainer() *SummaryContainer {
+func NewSummaryContainer(mapper *metricMapper) *SummaryContainer {
 	return &SummaryContainer{
 		Elements: make(map[uint64]prometheus.Summary),
+		mapper:   mapper,
 	}
 }
 
-func (c *SummaryContainer) Get(metricName string, labels prometheus.Labels, help string) (prometheus.Summary, error) {
+func (c *SummaryContainer) Get(metricName string, labels prometheus.Labels, help string, mapping *metricMapping) (prometheus.Summary, error) {
 	hash := hashNameAndLabels(metricName, labels)
 	summary, ok := c.Elements[hash]
 	if !ok {
+		quantiles := c.mapper.Defaults.Quantiles
+		if quantiles == nil {
+			fmt.Println("quantiles is nil")
```

Comment:
Is this a leftover debug statement or should an error be returned here?

Reply:
This and the print below are leftovers, apologies, I'm fixing those now with an updated commit in this PR.